src/main/java/com/playtika/homework2/task3/ManipulationsWithListOfStudentsInStream.java

```diff
@@ -0,0 +1,70 @@
+package com.playtika.homework2.task3;
+
+import com.playtika.homework2.task2.Student;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class ManipulationsWithListOfStudentsInStream {
+
+    public static void main(String[] args) {
+        int[] id = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+        String[] firstName = {"Андрей", "Павел", "Евгений", "Анастасия", "Юрий", "Ольга", "Александр", "Артем", "Денис", "Алиса"};
+        String[] lastName = {"Попков", "Жуковский", "Невмержицкий", "Бартош", "Хизюк", "Соболева", "Кондратович", "Петров", "Петров", "Иванова"};
+        int[] age = {21, 19, 22, 18, 17, 22, 19, 20, 20, 23};
+        List<Student> students = new ArrayList<>();
+
+        int countLastNames = lastName.length;
+        System.out.println("Список студентов:\n");
+        for (int i = 0; i < countLastNames; i++) {
+            students.add(new Student(id[i], firstName[i], lastName[i], age[i]));
+            System.out.println(students.get(i));
+        }
+
+        //    Метод для сортировки по возрасту
```

Comment:
move all separate logic blocks to separate methods in this class, invoke all these new methods from main method